package.json

```diff
@@ -111,7 +111,7 @@
     "ember-inflector": "^2.0.0"
   },
   "engines": {
-    "node": ">= 0.12.0"
```

Comment:
thanks for clarifying, sorry I wrote bad commit message, we still should keep `engines` section right ?